From 04ef398aed5114f64a7e91ce9ee0f417e22c69f7 Mon Sep 17 00:00:00 2001
From: Waldemar Glensk <waldemar.glensk@avnet.com>
Date: Wed, 17 Jun 2020 13:12:28 +0200
Subject: [PATCH] weston.ini. Disable panel. Cleanup.

---
 weston.ini.in | 21 +++++++--------------
 1 file changed, 7 insertions(+), 14 deletions(-)

diff --git a/weston.ini.in b/weston.ini.in
index 9e8bde4d..fd676d61 100644
--- a/weston.ini.in
+++ b/weston.ini.in
@@ -5,25 +5,18 @@ idle-time=0
 #xwayland=true
 #enable-overlay-view=1
 
-#[shell]
-#size=1920x1080
-
 [libinput]
 touchscreen_calibrator=true
 
-#[output]
-#name=HDMI-A-1
-#mode=1920x1080@60
-#transform=90
+[shell]
+panel-position=none
 
-#[output]
-#name=HDMI-A-2
-#mode=off
-#	WIDTHxHEIGHT    Resolution size width and height in pixels
-#	off             Disables the output
-#	preferred       Uses the preferred mode
-#	current         Uses the current crt controller mode
+[output]
+name=DSI-1
+#mode=1920x1080@60
 #transform=90
 
 [screen-share]
 command=@bindir@/weston --backend=rdp-backend.so --shell=fullscreen-shell.so --no-clients-resize
+
+
